backend/src/app.controller.ts

```diff
@@ -117,15 +117,24 @@ export class AppController {
     return await this.recommitmentService.checkRecommitmentPeriod();
   }
 
+  @Public()
   @Get('/chips')
-  async chips() {
+  @ApiQuery({
+    name: 'email',
+    type: String,
+    description: 'Email',
+    required: false
+  })
+  async chips(
+    @Query("email") email?: string,
+  ) {
     if (process.env.ENV !== 'dev') {
```

Comment:
Is this endpoint only for dev? No test or prod?

Reply:
Definitely not prod, this is really just for testing the chips endpoint. It shouldn't really be in anything at all eventually.